.golangci.yml

```diff
@@ -1,27 +1,18 @@
 linters:
   enable:
-    - staticcheck
-    - unconvert
+    - bodyclose
+    - dupword # Checks for duplicate words in the source code
```

Comment:
TIL 😄

Reply:
About the linter or about what it does? 😂  Yeah, comment may be a bit redundant; I took it from the containerd PR where I spotted that they fixed the issue

Reply:
about the linter - I never used this one before, so wasnt aware of it. Super useful!